integration-cli/docker_api_swarm_test.go

```diff
@@ -157,11 +156,18 @@ func (s *DockerSwarmSuite) TestUpdateSwarmAddExternalCA(c *check.C) {
 				Protocol: swarm.ExternalCAProtocolCFSSL,
 				URL:      "https://thishasnoca.org",
 			},
+			{
+				Protocol: swarm.ExternalCAProtocolCFSSL,
+				URL:      "https://thishasacacert.org",
+				CACert:   []byte("cacert"),
+			},
 		}
 	})
 	info, err := d1.SwarmInfo()
 	c.Assert(err, checker.IsNil)
-	c.Assert(info.Cluster.Spec.CAConfig.ExternalCAs, checker.HasLen, 1)
+	c.Assert(info.Cluster.Spec.CAConfig.ExternalCAs, checker.HasLen, 2)
+	c.Assert(info.Cluster.Spec.CAConfig.ExternalCAs[0].CACert, checker.IsNil)
+	c.Assert(reflect.DeepEqual(info.Cluster.Spec.CAConfig.ExternalCAs[1].CACert, []byte("cacert")), checker.True)
```

Comment:
`check.Equals` does not.   I didn't see `check.DeepEquals` though - will use that instead.